packages/shared/src/permissions/permissions.constants.ts

```diff
@@ -127,6 +128,17 @@ export const POLICY_PERMISSION_MAP: Record<Policy, Permission[]> = {
   TemplatesFullAccess: ["readData", "manageTemplates"],
   SqlExplorerFullAccess: ["readData", "runSqlExplorerQueries"],
   HoldoutsFullAccess: ["readData", "createAnalyses", "runQueries"],
+  ManageOfficialResources: [
+    "readData",
+    "createSegments",
+    "createMetrics",
+    "runQueries",
+    "manageFactTables",
+    "manageFactMetrics",
+    "manageFactFilters",
+    "createDimensions",
```

Comment:
I don't think we want/need all these dependencies.  Just have `manageOfficialResources` only (and maybe `readData`?).  This will allow more fine-grained control, since they can separately grant access to specific resource types as needed.

Reply:
@jdorn It feels like we should also give this policy the `runQueries` permission as well. Let me know if you disagree.

Reply:
There is no use case for someone being able to manage official metrics, but not non-official metrics.

However, there is a use case for someone being able to only edit metrics, but not fact tables for example.  If that person gets the "ManageOfficialResources" policy, they should still only be able to edit metrics, not fact tables.

Reply:
Gotcha. Thanks for that clarity. I think I've got a working solution that will be easy to maintain. Running through my manual test plan now.